code/modules/projectiles/projectile.dm

```diff
@@ -835,6 +835,8 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 /mob/living/carbon/xenomorph/projectile_hit(obj/projectile/proj, cardinal_move, uncrossing)
 	if(SEND_SIGNAL(src, COMSIG_XENO_PROJECTILE_HIT, proj, cardinal_move, uncrossing) & COMPONENT_PROJECTILE_DODGE)
 		return FALSE
+	if(HAS_TRAIT(src, TRAIT_BURROWED))
+		return
```

Comment:
Dont you already set passtable or whatever it is that lets bullets fly over? this wouldnt be needed then